Sources/Common/DataModel/Cell/test/testCell.js

```diff
@@ -47,13 +47,13 @@ test('Test vtkCell initialize with pointsIds', (t) => {
 
 test('Test vtkCell deepCopy', (t) => {
   const points = vtkPoints.newInstance();
-  points.setData([0, 0, 0, 2, 0, 0, 2, 2, 0]);
+  points.setData(Float64Array.from([0, 0, 0, 2, 0, 0, 2, 2, 0]));
```

Comment:
looking at the `vtkDataArray.setData` code, it seems that JS array (not typed) are supported.

If not, you need to update the TS doc